app/jsx/components/ArbitraryHTMLComp.jsx

```diff
@@ -12,10 +12,23 @@ export default class ArbitraryHTMLComp extends React.Component
     h1Level: PropTypes.number.isRequired
   }
 
+  // Attach script necessary for opening Depoit Wizard from any links defined as: http://open-deposit-wizard.com
+  componentWillMount() {
+    if (!(typeof document === "undefined")) {
+      const s = document.createElement("script")
+      s.async = true
+      s.innerHTML = "function openDepositWiz() {event.preventDefault(); document.getElementById(\'wizardlyDeposit\').click();}";
+      document.body.appendChild(s)
+    }
+  }
+
   render() {
     let origText = this.props.html
     let fixedText = origText.replace(/(<\/?[hH])([1-9]+)/g, 
                       (m, p1, p2) => p1 + (parseInt(p2) + this.props.h1Level - 1))
-    return <div dangerouslySetInnerHTML={{__html: fixedText}}/>
+    // Kludge for opening deposit wizard modal 
+    let fixedText2 = fixedText.replace(/<a href=\"http:\/\/open-deposit-wizard\.com\">/g, 
+      '<a href="" onClick="openDepositWiz();">')
+    return <div dangerouslySetInnerHTML={{__html: fixedText2}}/>
```

Comment:
Oh cool. Did this actually work and is there a page I can test it from?

Reply:
http://pub-jschol-dev.escholarship.org/uc/root/repository